vllm/executor/ray_gpu_executor.py

```diff
@@ -60,6 +60,14 @@ def _init_executor(self) -> None:
         # Create the parallel GPU workers.
         self._init_workers_ray(placement_group)
 
+    def shutdown(self) -> None:
+        if hasattr(self, "forward_dag") and self.forward_dag is not None:
```

Comment:
ideally this should be just handled by ray adag itself without manual cleanup.. can you create a github issue to ray?

Reply:
Sure, but just to understand the request: which part of cleanup were you referring to? `dag.teardown()` or killing the actors? Did you suggest to automatically tear down the DAG? Or kill the actors during DAG teardown? I think actors are not killed by default on purpose: actors are not (strictly) considered as DAG's resource, which allows them to be reused later.